lib/TypeNames/ModelTypeNames.cpp

```diff
@@ -407,17 +407,15 @@ void ptml::CTypeBuilder::printFunctionPrototype(const model::TypeDefinition
 }
 
 void ptml::CTypeBuilder::printSegmentType(const model::Segment &Segment) {
-  *Out << "\n" << getModelCommentWithoutLeadingNewline(Segment);
```

Comment:
Weird, you start with `"\n"` and then create a model comment without leading newline. Why not creating a comment with leading newline in the first place?

Reply:
`getModelComment` only adds new line _if_ there is a comment in there. That's what has been awkward about the whole situation.

And considering segments were always separated by an extra new-line, it looked awkward.